Gemfile

```diff
@@ -26,21 +25,24 @@ gem 'sidekiq-cron', '~> 1.9.1'
 gem 'aws-sdk-s3', '~> 1.94', require: false
 gem 'image_processing', '~> 1.12'
 gem 'active_storage_validations', '~> 1.1.3'
+gem "puma", ">= 5.0.0"
```

Comment:
Thanks 🙏🏼

I'd like to migrate to Puma for ages 😅

Reply:
Feline!